test/intl402/DateTimeFormat/prototype/formatRange/en-US.js

```diff
@@ -12,10 +12,10 @@ locale: [en-US]
 features: [Intl.DateTimeFormat-formatRange]
 ---*/
 
-const date1 = new Date("2019-01-03");
-const date2 = new Date("2019-01-05");
-const date3 = new Date("2019-03-04");
-const date4 = new Date("2020-03-04");
+const date1 = new Date("2019-01-03T00:00:00Z");
```

Comment:
Since you are checking the output value of the formatRange, please either use const date1 = new Date("2019-01-03T00:00") so the date object is created in local timezone not UTC. OR you specify the timezone of dtf in UTC so it will match. Otherwise, it won't be 1/3 in some timezone. Create the date1 ... date4 and your format output in the same time zone- either both in UTC or both in local timezone. See my comments in formatRangeToParts/en-US.js for more.